details/operator-sdk/controllers/astraconnector_controller.go

```diff
@@ -126,12 +126,7 @@ func (r *AstraConnectorController) Reconcile(ctx context.Context, req ctrl.Reque
 
 		// Stop reconciliation as the item is being deleted
 		// Do not requeue
-		return ctrl.Result{}, nil
-	}
-
-	if r.deployedAlready(astraConnector) {
```

Comment:
You should probably delete this function as well

Reply:
ahh good point